package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc",
     "watch": "tsc -w",
-    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "lint": "eslint . --ext .ts",
```

Comment:
You should pass a `source` folder to the `eslint` command that way you won't need the `.eslintignore` file

Reply:
removed .eslintignore, @VladNeacsu adding 'src' instead of '.' is fine, right?